_reports/macbox7.md

```diff
@@ -44,392 +44,392 @@ social:
 email: ajankit2304@gmail.com
 blog: https://macbox7.github.io/Blog-GSOC18/
 activity:
- - 0:
+ - 1:
```

Comment:
Your commit message does not reflect your changes. This file is not 2017 report but 2018.

Reply:
Also your commit message should follow the [commit guidelines](https://coala.io/commit).